### PR TITLE
Support shared files and directories

### PR DESCRIPTION
**Pull request by CristhianMotoche**
Solves #107 

## TODO:
- [ ] Is it necessary to have to separated entries for files and directories? At the end of the day the link is a file and it can be linked to anything (file or directory) that is in `/shared`.
- [x] Update documentation (`README.md`).
- [x] Prepare a new release (pvp in cabal file) and update `CHANGELOG.md`.

### CODE REVIEW

**Comment by CristhianMotoche** [comment]
@sestrella @juanpaucar Do you think it is necessary to have separated entries for files and directories? At the end of the day a symlink is a file that can be linked to anything (file or directory) that might be in `/shared`. Let me know your thoughts.

File: app/Main.hs [hunk 1]
```diff
@@ -153,6 +153,16 @@ main = do
                 let dpath = rpath </> destPath
                 (Hap.exec . Hap.MkDir . parent) dpath
                 Hap.scpDir srcPath dpath
+              forM_ configLinkedFiles $ \fileToLink -> do
```
Comment (sestrella) [inline app/Main.hs]: @CristhianMotoche could you please extract this code out to a separate function and add some tests:

**Comment by CristhianMotoche** [comment]
@sestrella @juanpaucar Could you please take another look?

File: hap.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+deploy_path: '/tmp/hapistrano'
```
[the remark this answers is not in the document]
Comment (CristhianMotoche) [reply hap.yaml]: Nope. My bad.

**Comment by CristhianMotoche** [comment]
@juanpaucar @sestrella Can you take another look, please?

**Pull request merged**